.github/workflows/CI.yml

```
@@ -31,10 +37,12 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
+        # https://github.com/actions/runner-images
         # macos-14==latest
-        # ubuntu-20.04==latest
-        os: ["ubuntu-latest", "ubuntu-24.04", "macos-14", "windows-latest"]
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        # ubuntu-24.04==latest
+        # windows-2022==latest
+        os: ["ubuntu-latest", "ubuntu-22.04", "macos-14", "windows-latest"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

If we were to follow [SPEC-0](https://scientific-python.org/specs/spec-0000/) we would have dropped 3.9 and 3.10 support by now. But for now, I'd consider trimming this to just the endpoints:

```suggestion
        python-version: ["3.9", "3.13"]
```

(because it seems to me unlikely that something would e.g. work on 3.9 but fail on 3.10 or 3.11)